### DISCUSSION TITLE
[Bug] я отпралив токен XCH с бижы Kucoin .  но на мой кошелек еще не поступили токены XCH

### DISCUSSION THREAD

**Discussion by Adoo997** [Chia-Network/chia-blockchain#16111] [Support]
### What happened?

![2023-08-18_09-30-23](https://github.com/Chia-Network/chia-blockchain/assets/142570244/2a75fc6f-9963-465b-9da0-6e171829290b)
![Screenshot_20230818_112742_KuCoin](https://github.com/Chia-Network/chia-blockchain/assets/142570244/90102a9a-9539-442a-9a4f-4e79cee6b9d5)

### Version

последнию

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by iron999** [comment]
добрый день?как ваши монеты,не пришли?я тоже на кукоине держу,думал отправить на кошелек и тут случайно наткнулся на ваше сообщение.и смотрю,помощи и ответа,ни от кого не видно!

**Comment by Adoo997** [comment]
ВОТ УЖЕ 3 ЧАСА  ПРОШЛО МОНЕТУ ЕЩЕ НЕ ПОСТУПИЛИ

**Comment by iron999** [comment]
а вы на кукоин писали?может из-за них проблема?напишите,пожалуйста,потом,чем закончится ситуация.

**Comment by Adoo997** [comment]
как и где можно переписоваться с поддерджкой  chia 
 Токены еще не пришли

**Comment by BrandtH22** [comment]
Привет, @Adoo997, обратите внимание на три вещи: 
1. Я преобразовал это в заявку в службу поддержки, а не в отчет об ошибке 
2. Как правило, мы можем обеспечить более немедленную и тщательную поддержку на нашем сервере Discord здесь https://discord.gg/chia 
3. Я использовал программное обеспечение для перевода этого ответа и приношу извинения за любые неточности.

Что касается монет, я бы сделал следующее: 
1. Скопируйте хэш tx из Kucoin и найдите его в проводнике, чтобы убедиться, что монеты попали на правильный адрес xch. Мне нравится использовать alltheblocks (https://alltheblocks.net/). Их поиск находится в правом верхнем углу. 
2. Если транзакция была подтверждена и находится в правильном адресе xch, я бы попробовал повторно синхронизировать кошелек chia. В настройках в разделе «Дополнительно» есть опция повторной синхронизации. 
3. Если транзакция не была подтверждена, я бы обратился в службу поддержки kucoin за дополнительной информацией.

original: 
```
Hey @Adoo997 , please note three things:
1. I have converted this to a support ticket rather than a bug report
2. We generally can provide more immediate and thorough support on our discord server here https://discord.gg/chia
3. I have used translation software for this response and I apologize for any inaccuracies. 

As for the coins, what I would do is:
1. Copy the tx hash from Kucoin and look it up on an explorer to ensure the coins have made it to the correct xch address. I like to use alltheblocks (https://alltheblocks.net/). Their search is in the top right corner.
2. If the transaction has been confirm and is in the correct xch address then I would try resyncing the chia wallet. In settings under advanced there is a resync option.
3. If the transaction has not been confirmed then I would reach out to kucoin support for more information.
```

**Reply by Mikael2121** [reply]
Добрый вечер BrandtH22,
По вашему комментарию видно,что вы очень хорошо разбираетесь в этом у меня похожая ситуация я отправил из Биржы проверил на alltheblock.net нашел мои сhis они на моем адресе,но у меня другая проблема и прошу при возможности помочь.мой кошелек Evergreen Mobile и он не показывает баланса а на Home Page -Sorry! An error occurred that we could not recover from.Я пробовал связаться с ними но ответа нету также писал на Discord пока безответно!Если ты можешь помочь я буду очень рад!

Good evening BrandtH22,
From your comment it is clear that you are very well versed in this. I have a similar situation, I sent it from the Exchange, checked on alltheblock.net, found my Chia, they are at my address, but I have a different problem and ask for help if possible. my Evergreen Mobile wallet and it Doesn't show balance on Home Page -Sorry! An error occurred that we could not recover from. I tried to contact them but there was no answer, I also wrote on Discord with no response yet! If you can help, I would be very glad!

**Reply by BrandtH22** [reply]
Hey @Mikael2121 , this sounds like a syncing issue with the wallet but unfortunately the evergreen app was developed and is managed by a third party so you will need to work with their team to determine what might be the cause.

Here is the evergreen discord server (https://discord.gg/sAZTy4DZJ7)

If you decide to use the official desktop chia wallet (https://www.chia.net/downloads/) then we can provide more support by reopening this ticket here or by joining us in our discord server (https://discord.gg/chia).

**Reply by Mikael2121** [reply]
Hey BrandtH22,thanks for the feedback, the guys from Evergreen confirmed that they have problems and I will wait, I wish you good luck in the new year. as soon as my problem is solved I will give an answer, maybe this will be useful for others who have problems in the future :)

**Reply by BrandtH22** [reply]
Hey @Mikael2121 , thank you for the confirmation and I am glad you were able to get in touch with their team. 
Thank you for the kind wishes and a Happy New Year to you also, may it be excellent and filled with great success!

I will keep an eye out for the resolution 👍

**Comment by WildPieChan** [comment]
Добрый день!
Подскажите, пожалуйста,  вы получили свои монеты?
Столкнулся с абсолютно такой же ситуацией
Переводил XCH с KuCoin на Evergreen wallet, уже прошло несколько дней, но монеты так и не появились
Поддержка не может помочь
![Screenshot_20240419_115944_Discord](https://github.com/Chia-Network/chia-blockchain/assets/102603398/5a28ff17-5233-4697-a550-41e51174e47a)
![Screenshot_20240419_115910_Discord](https://github.com/Chia-Network/chia-blockchain/assets/102603398/f6e23713-fed5-46af-9ce8-dab3b5a77e7f)

**Reply by Mikael2121** [reply]
Добрый день нет Evergreen не работает,скачайте в андроиде green app кошелек сгенерируйте сит фразу с Evergreen и ваши средств отображаться на новом кошельке.Удачи и хорошего дня!

**Reply by WildPieChan** [reply]
Золотой человек!
Спасибо тебе огромное!
Я уже перестал надеяться, что увижу свои токены
Ты лучший <3

**Comment by BrandtH22** [comment]
Hey all , this sounds like a syncing issue with the wallet but unfortunately the evergreen app was developed and is managed by a third party so you will need to work with their team to determine what might be the cause.

Here is the evergreen discord server (https://discord.gg/sAZTy4DZJ7)

If you decide to use the official desktop chia wallet (https://www.chia.net/downloads/) then we can provide more support by reopening this ticket here or by joining us in our discord server (https://discord.gg/chia).
Side note it sounds like the current workaround is to use the green wallet instead of the evergreen wallet.
